core/src/main/java/org/mule/routing/outbound/AbstractMessageSequenceSplitter.java

```diff
@@ -174,7 +178,12 @@ public Object transform(Object input)
         return resultEvents;
     }
 
-    private MuleMessage createMessage(Object payload, MuleMessage originalMessage)
+    protected void setMessageCorrelationId(MuleMessage message, String correlationId, int correlationSequence)
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "correlationSequence". ([squid:S1172](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1172))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7649&issues=AWi_Zq6votIXnmsUw8q7&open=AWi_Zq6votIXnmsUw8q7)

[ISSUE_KEY]: AWi_Zq6votIXnmsUw8q7